kinoml/core/kinase.py

```diff
@@ -0,0 +1,61 @@
+"""
+kinase_model.py
+Defines the Kinase class
+
+"""
+
+class Kinase(object):
```

Comment:
Hey, thanks for the submission! Do not consider this a code review, more like "I am leaving comments to myself while I catch up with the latest changes done while I was in vacations"!

I think this should be better integrated in the `kinoml.core.protein` object model; e.g. it could be a subclass of the whatever protein objects we end up using. I know this is just a placeholder for the information you need to store (could be a `dict`), so no worries for now.